src/database.rs

```diff
@@ -223,7 +220,7 @@ mod test {
         //non existing
         assert_eq!(db.load_prop("com.test", "/test", 1).await.unwrap(), None);
 
-        db.store_prop("com.test", "/test", &ser, 1).await.unwrap();
```

Comment:
`ser` is not used anymore and could be removed.

Reply:
Ser is still used at the end of the test